app/src/forms/form/exportService.js

```diff
@@ -184,10 +193,12 @@ const service = {
 
   _getSubmissions: (form, params, currentUser) => {
     service._checkPermission(currentUser, form.id, Permissions.SUBMISSION_READ, EXPORT_TYPES.submissions);
-    // possible params for this export include minDate and maxDate (full timestamp dates).
+    // params for this export include minDate and maxDate (full timestamp dates).
     return SubmissionData.query()
       .column(service._submissionsColumns(form, params))
       .where('formId', form.id)
+      .modify('filterDeleted', params.deleted)
+      .modify('filterDrafts', params.drafts)
```

Comment:
These modify functions should be added in after filterCreatedAt to stay alphabetically consistent.